### ISSUE TITLE
How to Disable new user login

### ISSUE THREAD

**Issue by vijayrajah** [cantino/huginn#1429]
I looked at the wiki, I wasn't able to find out how to do it.

I have a private instance, I would like to disable new user login, while retaining the ability to create new user using the admin screen.

is this possible?

-Thanks
Vijay

**Comment by dsander** [comment]
Hi @vijayrajah,

technically is it not possible to disable the login form, but if you configure the `INVITATION_CODE` in your `.env` file to be random and complex enough nobody will be able to create a new account.

**Comment by vijayrajah** [comment]
I know nothing about Ruby.. But, can we not, just remove the button from the login page?

**Comment by dsander** [comment]
You can remove it, but I do not understand how it would make a difference. The login page already shows a message that the Huginn installation is not open to the public.

**Comment by meetai** [comment]
Agreed that there should be a way to disable this functionality, but at the same time, the `.env` trick could make up for your wish in the meantime.
